backend/config/curation-api.yml

```diff
@@ -496,16 +496,14 @@ components:
         assets:
           type: array
           items:
-            type: object
-            properties:
-              presigned_url:
-                type: string
-              file_name:
-                type: string
-              file_size:
-                type: number
-              filetype:
-                type: string
+            allOf:
+              - $ref: "#/components/schemas/dataset_asset"
```

Comment:
@Bento007 I was surprised your asset endpoint was passing the schema validation (due to cxg). Added this so it gets the benefit of the enum def validation